### PR TITLE
Expose OQL.genes to window

### PR DESCRIPTION
**Pull request by adamabeshouse**

### CODE REVIEW

File: src/globalComponents.tsx [hunk 1]
```diff
@@ -15,6 +16,9 @@ class GlobalStores {
 }
 
 (window as any).getStudySummaryUrl = getStudySummaryUrl;
+(window as any).oqlGenes = (oqlQuery:string)=>{
```
Comment (inodb) [inline src/globalComponents.tsx]: @adamabeshouse Prolly not in this PR. But would be nice to expose these in a namespaced manner. E.g. `window.frontendVars.oqlGenes` similar for other variables/functions. Makes it a little easier to see in backend project where tight integration happens

Comment (adamabeshouse) [reply src/globalComponents.tsx]: good idea

**Comment by alisman** [comment]
should this go to rc though?

**Comment by adamabeshouse** [comment]
@alisman I thought master since its kind of a bug... but rc fine too. whatever you think

**Comment by fedde-s** [comment]
I ran into the issue when trying to implement a new feature, so for that I'll only need it in `rc` for now. I'll probably have time to check if it fixes the specific problem I was having in the next few days, but I don't mind if it's already been merged into `rc` by then, in which case I'll just address any follow-up issues iteratively.

File: src/shared/lib/oql/oqlfilter.js [hunk 2]
```diff
@@ -1,4 +1,4 @@
-// Requires parser to be window object, window.oql_parser
+// Requires parser to be window object, oql_parser
```
Comment (fedde-s) [inline src/shared/lib/oql/oqlfilter.js]: I still find this comment a bit confusing. Does it actually require anything from the window object, or does our tooling (Webpack, maybe helped by Babel?) just make the import statement below work?

Comment (alisman) [reply src/shared/lib/oql/oqlfilter.js]: no this is not related to tooling.  it's basically just saying, this is an expected global

Comment (alisman) [reply src/shared/lib/oql/oqlfilter.js]: actually this comment is defunct, right? b/c we now import it below.  strike comment?

Comment (adamabeshouse) [reply src/shared/lib/oql/oqlfilter.js]: heh this is the result of a find and replace, this comment should be deleted. good catch

**Pull request merged**